components/org.wso2.carbon.identity.api.server.organization.management/org.wso2.carbon.identity.api.server.organization.management.v1/src/main/java/org/wso2/carbon/identity/api/server/organization/management/v1/service/OrganizationManagementService.java

```diff
@@ -727,7 +727,7 @@ private OrganizationsResponse getOrganizationsResponse(Integer limit, String aft
                 Collections.reverse(organizations);
             }
             if (!isFirstPage) {
-                String encodedString = Base64.getEncoder().encodeToString(organizations.get(0).getCreated()
+                String encodedString = Base64.getEncoder().encodeToString(organizations.get(0).getCreated().toString()
```

Comment:
Remove .toString()

Reply:
This is needed as the `created` attribute in the Organization class is an Instant object and needs to be converted to a string.